WooCommerce/WooCommerceUITests/Tests/OrdersTests.swift

```diff
@@ -0,0 +1,20 @@
+import UITestsFoundation
+import XCTest
+
+final class OrdersTests: XCTestCase {
+    override func setUp() {
+        continueAfterFailure = false
+
+        // UI tests must launch the application that they test.
+        let app = XCUIApplication()
+        app.launchArguments = ["logout-at-launch", "disable-animations", "mocked-wpcom-api", "-ui_testing"]
+        app.launch()
+    }
+
+    // TODO: Write real test, this is a placeholder for now
+    func testGotoOrdersScreen() throws {
+        try LoginFlow.login()
```

Comment:
(Keeping in mind that this is a temporary and unfinished test) Generally, the login is a part of `setUp`. In cases if it's not, it's natural to include the logout into the `tearDown`, to make sure the test leaves the app in the state before the test.